site/docs/pages/index.mdx

```diff
@@ -68,29 +79,93 @@ bun add @coinbase/onchainkit
 :::
 
       </div>
-      <a href="/getting-started" className="font-sans justify-self-end border rounded-xl border-solid border-white px-4 py-3 font-bold">
-        View docs 
+      <a href="/getting-started" className="font-sans justify-self-end border rounded-xl border-solid border border-indigo-400 text-indigo-400 px-4 py-3 font-bold" target="_blank">
```

Comment:
@abcrane123 as follow up, let's make sure to remove the `target="_blank"`, after all those are internal links and it should not open a new up.